### PR TITLE
Add contribution guidelines

### PR DESCRIPTION
**Pull request by eps90**
@jhuet @andres-montanez 
I'm adding the contribution guidelines for developers who will make future Pull Requests. I've added some notes about code style, branching, testing and so on. GitHub supports contributing guidelines as described [here](https://help.github.com/articles/setting-guidelines-for-repository-contributors/). 
I hope this will make the Magallanes better app! :)

Please review it and tell me what do you think. All feedback is welcome!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+2. **PHP Mess Detector**
+3. PHP Copy/Paste Detector
+4. PHP Dead Code Detector
+
+## Testing and quality
+We use PHPUnit to test our classes. Now not the whole project is covered with tests but we've been working on it for some time. If you want your code to be merged into Magallanes, we want you to push it with proper tests. We would love to reach and keep at leats 90% of code coverage. In short time we want to configure quality tools to make sure your code is tested properly with minimum coverage. Anyway, try to keep 100% of Code Coverage in your pull requests.
+Tests structure follow the same structure as production code with `Test` suffix in class and file name. All tests should go to `tests` directory in project root.  So if you've created a class `Mage\Tasks\BuilIn\NewTask` the testing class should be called `MageTest\Tasks\BuiltIn\NewTaskTest`. 
+
+## Configuration
+Magallanes configuration is kept in YAML files. Please follow those rules whie adding or changing the configuration:
```
Comment (DaniSancas) [inline CONTRIBUTING.md]: Typo detected: (...) those rules **whie** adding (...)

Comment (eps90) [inline CONTRIBUTING.md]: @DaniSancas Thanks for spotting :) Just fixed

**Comment by eps90** [comment]
I've just added some instructions about code coverage in our project. There are also some good practices in writing commit message.

**Comment by andres-montanez** [comment]
Love it!

**Comment by eps90** [comment]
Thanks! ;)

**Pull request merged**